src/Microsoft.ML.Transforms/Text/LdaStaticExtensions.cs

```diff
@@ -0,0 +1,173 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.TextAnalytics;
+using Microsoft.ML.StaticPipe;
+using Microsoft.ML.StaticPipe.Runtime;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.ML.Transforms.Text
+{
+    /// <summary>
+    /// Information on the result of fitting a LDA transform.
+    /// </summary>
+    /// <typeparam name="T">The type of the values.</typeparam>
+    public sealed class LdaFitResult<T>
```

Comment:
>T [](start = 37, length = 1)

why is `T` necessary?

Reply:
fixed

---
In reply to: [232437603](https://github.com/dotnet/machinelearning/pull/1410#discussion_r232437603) [](ancestors = 232437603)